invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/grid.overrides

```diff
@@ -181,3 +181,10 @@
     }
   }
 }
+
+.ui.stackable.grid > .column:not(.row){
```

Comment:
this change overrides all the stackable grids padding across the theme. Was it tested to see if it causes any changes elsewhere, since it is affecting the whole theme?

Reply:
You're right. I will specify the class!